src/internal/observable/fromEvent.ts

```diff
@@ -22,8 +22,8 @@ export interface NodeCompatibleEventEmitter {
 }
 
 export interface JQueryStyleEventEmitter {
-  on: (eventName: string, handler: Function) => void;
-  off: (eventName: string, handler: Function) => void;
+  on: (eventName: string, handler: (event: any) => any) => void;
+  off: (eventName: string, handler: (event: any) => any) => void;
```

Comment:
issue: jQuery listener signatures look like [this](https://github.com/ReactiveX/rxjs/blob/9e9064b7513aec4c365e792beac211f9d12bd7b9/spec-dtslint/observables/fromEvent-spec.ts#L17-L23). See also #5584 - which is gonna need some rebasing, now.

Reply:
Must've been mangled in my other work. Thanks for catching that.

Reply:
Oh, I see, that's in the dtslint test.

I've made the fix and uncommented the test, updating it a little.